storage/aptosdb/src/db/fake_aptosdb.rs

```diff
@@ -767,6 +772,23 @@ impl DbReader for FakeAptosDB {
         )
     }
 
+    fn get_account_all_transaction_summaries(
```

Comment:
nit: "all" seems redundant and confusing here, `get_account_transaction_summaries` looks fine to me.

Reply:
Done.